iron-bank-suite/tornado-cash-user/src/agent.js

```diff
@@ -0,0 +1,69 @@
+const { Finding, FindingSeverity, FindingType } = require("forta-agent")
+const axios = require("axios")
+const { markets } = require("./iron-bank-markets")
+
+const marketsAddresses = Object.values(markets)
+
+const tornadoCashAddresses = [
+  '0x12D66f87A04A9E220743712cE6d9bB1B5616B8Fc', // 0.1 ETH
+  '0x47CE0C6eD5B0Ce3d3A51fdb1C52DC66a7c3c2936', // 1 ETH
+  '0x910cbd523d972eb0a6f4cae4618ad62622b39dbf', // 10 ETH
+  '0xA160cdAB225685dA1d56aa342Ad8841c3b53f291', // 100 ETH
+]
+
+// We cannot put the key in a config file
+// because it would be easy to extract it
+const apiKey = "test"
```

Comment:
I guess the idea is to put the api key here before obfuscating the agent code and deploying it, right?